2.7/s2i/bin/assemble

```diff
@@ -14,6 +14,9 @@ echo "---> Copying application source ..."
 cp -Rf /tmp/src/. ./
 
 if [[ -f requirements.txt ]]; then
+  if [ -n "$PIP_INDEX_URL" ]; then
+    export PIP_INDEX_URL=${PIP_INDEX_URL}
```

Comment:
what's the point of this export, given that the value is already set and you're then setting it to the same value again?